Kudu.Services.Test/Jobs/InvokeTriggeredJobReadonlyFileSystemTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Kudu.Contracts.Jobs;
+using Kudu.Contracts.Tracing;
+using Kudu.Core.Infrastructure;
+using Kudu.Services.Jobs;
+using Kudu.TestHarness.Xunit;
+using Moq;
+using System;
+using System.IO.Abstractions;
+using System.Net;
+using System.Net.Http;
+using Xunit;
+
+namespace Kudu.Services.Test.Jobs
+{
+    [KuduXunitTestClass]
+    public class InvokeTriggeredJobReadonlyFileSystemTests
```

Comment:
right now tests are run in parallel per class, i think it is prefer to create more class than putting multiple test within same class.

JobsController is under "Kudu.Services" that is why i create similar folder structure. 
i can name this new class start with JobsController, to make it more easier to related to JobsController.cs